plugins/module_utils/foreman_helper.py

```diff
@@ -592,10 +664,11 @@ def connect(self):
 
         self.foremanapi = apypie.Api(
             uri=self._foremanapi_server_url,
-            username=to_bytes(self._foremanapi_username),
-            password=to_bytes(self._foremanapi_password),
```

Comment:
this was done as part of https://github.com/theforeman/foreman-ansible-modules/pull/609, but it seems ansible's `Request` does that for us instead.